external-crates/move/crates/move-analyzer/src/symbols/ide_strings.rs

```diff
@@ -144,6 +144,7 @@ pub fn type_to_ide_string(sp!(_, t): &Type, verbose: bool) -> String {
             )
         }
         Type_::Anything => "_".to_string(),
+        Type_::Void => "_".to_string(),
```

Comment:
```suggestion
        Type_::Void => "!".to_string(),
```
🤷 
Though maybe it is best to not ever display it as such? at least until it is a real type